src/Preprocessor.zig

```diff
@@ -893,6 +1281,45 @@ pub fn expandedSlice(pp: *Preprocessor, tok: Token) []const u8 {
     return tmp_tokenizer.buf[res.start..res.end];
 }
 
+fn pasteTokens2(pp: *Preprocessor, lhs: []const Token, rhs: []const Token) Error![]const Token {
```

Comment:
This doesn't seem at all correct, as far as I understand pasting is supposed to form a single token without doing any expansion.

Reply:
Yeah, it's wrong, I need to revisit this.